apps/frontend/src/components/generic/ApexLineChart.vue

```diff
@@ -17,10 +17,9 @@ import {ApexOptions} from 'apexcharts';
 import {Category} from './ApexPieChart.vue';
 import {Prop} from 'vue-property-decorator';
 
-let id_counter = 0;
+let idCounter = 0;
 function next_id(): number {
-  id_counter += 1;
-  return id_counter;
+  return idCounter++;
```

Comment:
This seems to be raising another Sonarcloud error. This code isn't actually used anywhere once the line below is removed.